webapp-django/crashstats/settings/bundles.py

```diff
@@ -430,22 +433,22 @@
         for asset_file in v['source_filenames']:
             if asset_file in _used:
                 # Consider using warnings.warn here instead
-                print '{:<52} in {:<20} already in {}'.format(
+                print('{:<52} in {:<20} already in {}'.format(
                     asset_file,
                     k,
                     _used[asset_file]
-                )
+                ))
                 _trouble.add(asset_file)
             _used[asset_file] = k
 
     for asset_file in _trouble:
-        print "REPEATED", asset_file
+        print("REPEATED", asset_file)
         found_in = []
         sets = []
         for k, v in config.items():
             if asset_file in v['source_filenames']:
                 found_in.append(k)
                 sets.append(set(list(v['source_filenames'])))
-        print "FOUND IN", found_in
-        print "ALWAYS TOGETHER WITH", set.intersection(*sets)
+        print("FOUND IN", found_in)
+        print("ALWAYS TOGETHER WITH", set.intersection(*sets))
```

Comment:
You can't do this with the print function. You'll have to change this to:

```
print('FOUND IN %s' % found_in)
print('ALWAYS TOGETHER WITH %r' % set.intersection(*sets))
```

or something like that.